{
  "comments": [
    {
      "key": {
        "uuid": "3fa7e38b_710b517c",
        "filename": "doc/source/conf.py",
        "patchSetId": 3
      },
      "lineNbr": 193,
      "author": {
        "id": 13252
      },
      "writtenOn": "2019-09-26T08:34:02Z",
      "side": 1,
      "message": "Hmm, that bug is resolved, so why is this still needed?",
      "revId": "4330fc4afe79ac0fef5198d129de8f124bbafd65",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    }
  ]
}